jwt-authorizer/src/layer.rs

```diff
@@ -209,29 +266,35 @@ where
     type Future = BoxFuture<'static, Result<Request<B>, AuthError>>;
 
     fn authorize(&self, mut request: Request<B>) -> Self::Future {
-        let authorizer = self.auth.clone();
-        let h = request.headers();
+        let tkns_auths: Vec<(Option<String>, Arc<Authorizer<C>>)> = self
+            .auths
+            .iter()
+            .map(|a| (a.extract_token(request.headers()), a.clone()))
```

Comment:
This could be a `filter_map` instead so you end up with an empty array if there are no tokens found rather then having to juggle with Options in the tuples later. Though still not sure the multiple jwt tokens in the first place :)

Reply:
Yeah, you're right! I'll fix this.